### PR TITLE
Enhance tests

### PR DESCRIPTION
**Pull request by jcaesar**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -8,3 +8,7 @@ license = "Apache-2.0"
 repository = "https://github.com/sourcefrog/unix_mode"
 
 [dependencies]
+
+[target.'cfg(unix)'.dependencies]
```
Comment (sourcefrog) [inline Cargo.toml:12]: I think we can make this `dev-dependencies` as it's only needed for tests?

Comment (sourcefrog) [reply Cargo.toml]: done.

Comment (jcaesar) [reply Cargo.toml]: Oh whoops! Thanks for catching that.

**Comment by sourcefrog** [comment]
These are both in a new release, https://crates.io/crates/unix_mode/0.1.2

**Pull request merged**